tsnet/tsnet.go

```diff
@@ -151,6 +152,9 @@ func (s *Server) Close() error {
 		}()
 	}
 
+	if s.netstack != nil {
+		s.netstack.Close()
```

Comment:
s.netstack = nil ?

(I don't expect this to actually matter, nothing should try to use this struct again)

Reply:
sure. done.